### PR TITLE
EMTF NN version update

### PR DESCRIPTION
**Pull request by eyigitba**
#### PR description:

This PR updates the version of the data file for the EMTF displaced NN `v10.1`. The previous version is outdated, and needs to replaced with the version available in firmware at P5. Some performance changes are expected with respect to the old version `v5`

#### PR validation:

Validated with `runTheMatrix.py -l limited -i all --ibeos`. 

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

This change requires https://github.com/cms-data/L1Trigger-L1TMuon/pull/22 to work

### CODE REVIEW

**Comment by epalencia** [comment]
Please test

**Comment by eyigitba** [comment]
This PR requires https://github.com/cms-data/L1Trigger-L1TMuon/pull/22 to work. Can you please add that to the testing parameters? Thanks!

**Comment by cecilecaillol** [comment]
please test with https://github.com/cms-data/L1Trigger-L1TMuon/pull/22

**Comment by cecilecaillol** [comment]
+l1

File: L1Trigger/L1TMuonEndCap/src/PtAssignmentEngineDxy.cc [hunk 1]
```diff
@@ -21,8 +21,8 @@ void PtAssignmentEngineDxy::configure(int verbose, const std::string pbFileNameD
   pbFileNameDxy_ = pbFileNameDxy;
   std::string pbFilePathDxy_ = "L1Trigger/L1TMuon/data/emtf_luts/" + pbFileNameDxy_;
 
-  inputNameDxy_ = "batch_normalization_1_input";
-  outputNamesDxy_ = {"dense_4/BiasAdd"};
+  inputNameDxy_ = "input1";
```
Comment (perrotta) [inline L1Trigger/L1TMuonEndCap/src/PtAssignmentEngineDxy.cc:24]: Are changes in this file related to the update of the EMTF displaced NN?

Comment (eyigitba) [reply L1Trigger/L1TMuonEndCap/src/PtAssignmentEngineDxy.cc:24]: Yes. The names of the input and output nodes changed in the new protobuf file. The changes are made to adapt to that.

**Comment by perrotta** [comment]
+1

**Pull request merged**